src/components/forms/DatePicker/DatePicker.stories.tsx

```diff
@@ -61,7 +61,8 @@ export const completeDatePicker = (
       <DatePicker
         id="appointment-date"
         name="appointment-date"
-        aria-describedby="appointment-date-label appointment-date-hint"
+        aria-describedby="appointment-date-hint"
+        aria-labelledby="appointment-date-label"
```

Comment:
ref: https://github.com/uswds/uswds/compare/v2.12.2...v2.13.0#diff-85ddb26937b3016aaddc3058619885bb9d66a2c0439d2162b262ab1f236e1349R17

Reply:
The links aren't working for me, but I was able to find and confirm what I think you're pointing to: 
<img width="1228" alt="image" src="https://user-images.githubusercontent.com/15805554/176021929-f858c378-0505-4cf7-b9bd-3b16289d866c.png">